### DISCUSSION TITLE
1.20.6 Support

### DISCUSSION THREAD

**Discussion by Thomas7191** [kous500/CurveBuilding#22] [General]
Hello, when will version 1.20.6 be available for the server?

**Comment by kous500** [comment]
The new version of the plugin is now supported in version 1.20.6.

https://github.com/kous500/CurveBuilding/releases/tag/0.6.2

**Reply by WOSK24** [reply]
It doesn't seem to work for FAWE I'm trying to //pos set and it is not working

**Reply by kous500** [reply]
As of version 0.6, the //pos command has been renamed to //bcpos to avoid conflicts with WorldEdit.
